### PR TITLE
fix: links and references to Atala in the quick-start.md guide

### PR DESCRIPTION
**Pull request by todorkoleviohk**
https://input-output.atlassian.net/browse/ATL-7064

This is a non-conflicting merge of changes made before some other changes were merged from other branches.
Please let's merge this one and then we look into the other open PR's, merge and then final overhaul to make sure all fixes are in and everything is correct.

I have an automated tool to check that all links work and all replacement table is applied.

### CODE REVIEW

**Review by yshyn-iohk — APPROVED** [review]

**Review by petevielhaber — APPROVED** [review]

File: src/components/resources/index.js [hunk 1]
```diff
@@ -129,13 +129,13 @@ const RESOURCES = [
         title: "Wallet SDK Swift",
         Svg: Graphics.walletSdkSwift,
         content: "Build Swift-based applications using our SDKs",
-        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/atalaprismsdk/"
```
Comment (mkbreuningIOHK) [inline src/components/resources/index.js:132]: Could you explain why changing the path? The swift SDK is working well in the current staging website. It was changed to https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/edgeagentsdk/ in the PR https://github.com/input-output-hk/atala-prism-docs/pull/93

File: documentation/docs/sidebars.js [hunk 2]
```diff
@@ -49,11 +49,11 @@ const sidebars = {
         'atala-prism/overview',
         {
           type: 'category',
-          label: 'PRISM Cloud Agent',
+          label: 'Cloud Agent',
           collapsed: true,
           link: {
             type: 'generated-index',
-            title: 'PRISM Cloud Agent',
+            title: 'Cloud Agent',
```
Comment (mkbreuningIOHK) [inline documentation/docs/sidebars.js:56]: And `Atala PRISM Cloud Agent` to be changed to `Identus Cloud Agent` or `Cloud Agent`

File: src/components/resources/index.js [hunk 3]
```diff
@@ -129,13 +129,13 @@ const RESOURCES = [
...
     },
     {
         title: "Wallet SDK TypeScript",
         Svg: Graphics.walletSdkTS,
         content: "Build web-based applications using our SDKs",
-        linkTo: "#"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-ts/"
```
Comment (mkbreuningIOHK) [inline src/components/resources/index.js:138]: Please explain the reason of this link change. So we are sure it is not breaking something somewhere else.

File: src/components/resources/index.js [hunk 4]
```diff
@@ -129,13 +129,13 @@ const RESOURCES = [
         title: "Wallet SDK Swift",
         Svg: Graphics.walletSdkSwift,
         content: "Build Swift-based applications using our SDKs",
-        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/atalaprismsdk/"
     },
     {
         title: "Wallet SDK TypeScript",
```
Comment (mkbreuningIOHK) [inline src/components/resources/index.js:135]: to `Edge Agent SDK TypeScript` or `Edge Agent SDK TS`

File: src/components/resources/index.js [hunk 5]
```diff
@@ -129,13 +129,13 @@ const RESOURCES = [
...
         Svg: Graphics.walletSdkTS,
         content: "Build web-based applications using our SDKs",
-        linkTo: "#"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-ts/"
     },
     {
         title: "Wallet SDK KMM",
```
Comment (mkbreuningIOHK) [inline src/components/resources/index.js:141]: To `Edge Agent SDK KMP`
And `Build KMP-based`

File: documentation/docs/quick-start.md [hunk 6]
```diff
@@ -54,7 +54,7 @@ More in-depth documentation about the different Wallet SDKs can be found here ([
 More in-depth documentation about PRISM Mediator can be found [here](/docs/atala-prism/prism-mediator).
 
 #### A Node for a Verifiable Data Registry (VDR)
-To issue and verify VCs to and from DIDs, we need a [Verifiable Data Registry (VDR)](/docs/concepts/glossary/#verifiable-data-registry) that is globally resolvable and always on. In PRISM's case, it is `prism-node`, [anchoring](/docs/concepts/glossary/#anchoring) key information required for issuance and verification on the Cardano blockchain.
+To issue and verify VCs to and from DIDs, we need a [Verifiable Data Registry (VDR)](/docs/concepts/glossary/#verifiable-data-registry) that is globally resolvable and always on. In PRISM's case, it is `prism-node`, [anchoring](/docs/concepts/glossary/#anchoring) key information required for issuance and verification on the Distributed Ledger blockchain.
```
Comment (mkbreuningIOHK) [inline documentation/docs/quick-start.md:57]: @todorkoleviohk , in the Quick Start Guide, there are more to update: the links to the old repo hyperledger lab, the Wallet to Edge Agent.
I advise you to review what @CryptoKnightIOG has done in this PR https://github.com/input-output-hk/atala-prism-docs/pull/96 and work together to finish it.

**Review by mkbreuningIOHK — CHANGES_REQUESTED** [review]
Quick Start Guide renaming is not finished. Please align with @CryptoKnightIOG and his PR https://github.com/input-output-hk/atala-prism-docs/pull/96. And pair together to finish this.

File: src/components/resources/index.js (hunk 1 above)
Comment (CryptoKnightIOG) [reply src/components/resources/index.js:132]: reverted back to original link, we will do a link check in second iteration.

File: src/components/resources/index.js (hunk 3 above)
Comment (CryptoKnightIOG) [reply src/components/resources/index.js:138]: reverted back to original link, we will do a link check in second iteration.

File: src/components/resources/index.js (hunk 4 above)
Comment (CryptoKnightIOG) [reply src/components/resources/index.js:135]: fixed

File: src/components/resources/index.js (hunk 5 above)
Comment (CryptoKnightIOG) [reply src/components/resources/index.js:141]: fixed

File: src/components/resources/index.js (hunk 1 above)
Comment (todorkoleviohk) [reply src/components/resources/index.js]: https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/
does not work

https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/edgeagentsdk/ 
Looks fine :)

File: src/components/resources/index.js (hunk 3 above)
Comment (todorkoleviohk) [reply src/components/resources/index.js]: # must have been a placeholder from early days before the repo was actually ready :)
This doesn't lead to anywhere and changing it to a good link does not incur a risk of breaking.

**Comment by todorkoleviohk** [comment]
Had to open a new one as this one was far too much behind and was messy already.
https://github.com/input-output-hk/atala-prism-docs/pull/100

**Pull request closed without merging**